config/packages/twig.yaml

```diff
@@ -1,9 +1,8 @@
 twig:
-    default_path: '%kernel.project_dir%/templates'
```

Comment:
I removed this line as it was also remove from the recipe

https://github.com/symfony/recipes/blob/main/symfony/twig-bundle/6.4/config/packages/twig.yaml